### PR TITLE
[4.1] Remove FLoC

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #36853 

Remove useless code

### CODE REVIEW

File: installation/sql/mysql/supports.sql [hunk 1]
```diff
@@ -278,10 +278,6 @@ INSERT INTO `#__postinstall_messages` (`extension_id`, `title_key`, `description
...
-SELECT `extension_id`, 'PLG_SYSTEM_HTTPHEADERS_POSTINSTALL_INTRODUCTION_TITLE', 'PLG_SYSTEM_HTTPHEADERS_POSTINSTALL_INTRODUCTION_BODY', 'PLG_SYSTEM_HTTPHEADERS_POSTINSTALL_INTRODUCTION_ACTION', 'plg_system_httpheaders', 1, 'action', 'site://plugins/system/httpheaders/postinstall/introduction.php', 'httpheaders_postinstall_action', 'site://plugins/system/httpheaders/postinstall/introduction.php', 'httpheaders_postinstall_condition', '4.0.0', 1 FROM `#__extensions` WHERE `name` = 'files_joomla';
-INSERT INTO `#__postinstall_messages` (`extension_id`, `title_key`, `description_key`, `action_key`, `language_extension`, `language_client_id`, `type`, `action_file`, `action`, `condition_file`, `condition_method`, `version_introduced`, `enabled`)
-SELECT `extension_id`, 'COM_ADMIN_POSTINSTALL_MSG_FLOC_BLOCKER_TITLE', 'COM_ADMIN_POSTINSTALL_MSG_FLOC_BLOCKER_DESCRIPTION', '', 'com_admin', 1, 'message', '', '', '', '', '3.9.27', 1 FROM `#__extensions` WHERE `name` = 'files_joomla';
```
Comment (nikosdion) [inline installation/sql/mysql/supports.sql:284]: We should add a DELETE FROM in a new SQL update file for existing sites which already have the post–installation message.

**Review by nikosdion — CHANGES_REQUESTED** [review]
Just a small change necessary to remove the Post-Installation Message on existing sites which already have the message installed. Otherwise resetting the messages would show two problem language strings.

File: installation/sql/mysql/supports.sql (hunk 1 above)
Comment (brianteeman) [reply installation/sql/mysql/supports.sql:284]: i already did

Comment (brianteeman) [reply installation/sql/mysql/supports.sql:284]: I just forgot to commit the new files

File: installation/sql/mysql/supports.sql [hunk 2]
```diff
@@ -278,10 +278,6 @@ INSERT INTO `#__postinstall_messages` (`extension_id`, `title_key`, `description
...
-INSERT INTO `#__postinstall_messages` (`extension_id`, `title_key`, `description_key`, `action_key`, `language_extension`, `language_client_id`, `type`, `action_file`, `action`, `condition_file`, `condition_method`, `version_introduced`, `enabled`)
-SELECT `extension_id`, 'PLG_SYSTEM_HTTPHEADERS_POSTINSTALL_INTRODUCTION_TITLE', 'PLG_SYSTEM_HTTPHEADERS_POSTINSTALL_INTRODUCTION_BODY', 'PLG_SYSTEM_HTTPHEADERS_POSTINSTALL_INTRODUCTION_ACTION', 'plg_system_httpheaders', 1, 'action', 'site://plugins/system/httpheaders/postinstall/introduction.php', 'httpheaders_postinstall_action', 'site://plugins/system/httpheaders/postinstall/introduction.php', 'httpheaders_postinstall_condition', '4.0.0', 1 FROM `#__extensions` WHERE `name` = 'files_joomla';
```
Comment (richard67) [inline installation/sql/mysql/supports.sql]: Why are line 281 and 282 deleted here?

File: htaccess.txt [hunk 3]
```diff
@@ -28,14 +28,6 @@ Options -Indexes
...
 <IfModule mod_headers.c>
 	Header always set X-Content-Type-Options "nosniff"
-##
-# Disable Federated Learning of Cohorts (FLoC)
-# If you uncomment the below directive you have to allow this technology in the
-# Global Configuration of Joomla. Read more about this in the Post-Installation
-# message in the backend.
-##
-# Header always set Permissions-Policy "interest-cohort=()"
-</IfModule>
```
Comment (Hackwar) [inline htaccess.txt:38]: I doubt that this line should be removed as well. 😃

File: installation/sql/mysql/supports.sql (hunk 2 above)
Comment (nikosdion) [reply installation/sql/mysql/supports.sql]: I think @brianteeman got confused here. The HTTP Headers plugin is unrelated to Block FLoC.

The thing is that this Global Configuration (application level!) option is implemented in that _optional_ system plugin (defying all logic) which is probably what led to the confusion here.

Comment (richard67) [reply installation/sql/mysql/supports.sql]: I think he just accidentally removed those rows.

**Review by richard67 — APPROVED** [review]

**Review by wilsonge — APPROVED** [review]

**Comment by brianteeman** [comment]
@zero-24 should I also backport this to j3?

**Comment by zero-24** [comment]
Yes please, from my understanding it should be merged up from 3.10-dev anyway.

**Comment by brianteeman** [comment]
I will do a pr for 3.10 as well. Its a different pr so merging up wouldnt really help

**Comment by zero-24** [comment]
Thanks :)

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ff448095933099aa369c1321cafb4d5743d1f236<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36855">issues.joomla.org/tracker/joomla-cms/36855</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on ff448095933099aa369c1321cafb4d5743d1f236<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36855">issues.joomla.org/tracker/joomla-cms/36855</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36855">issues.joomla.org/tracker/joomla-cms/36855</a>.</sub>

**Comment by bembelimen** [comment]
I've used the upmerged fix in #37104 
Thanks for the removal!

**Pull request closed without merging**